src/pages/Faucet/index.tsx

```diff
@@ -61,42 +61,34 @@ const faucetTokens = [
   {
```

Comment:
Can't you import this data from `src/constants/tokens.ts` and use this? Would remove some redundancy and make it easier to update addresses :D